autoware_reference_system/include/autoware_reference_system/system/timing/benchmark.hpp

```diff
@@ -115,6 +129,12 @@ struct BenchmarkCPUUsage
   static constexpr time_t LANELET_2_GLOBAL_PLANNER = milliseconds(0);
   static constexpr time_t LANELET_2_MAP_LOADER = milliseconds(0);
 
+  static constexpr time_t POINT_CLOUD_FUSION_MAX_INPUT_TIME_DIFF = seconds(9999);
```

Comment:
@elfenpiff we should probably add this to [the fusion node requirements section](https://github.com/elfenpiff/reference-system/tree/add-fusion-node-max-input-timediff/autoware_reference_system#reference-system-autoware) in the `README`